src/QMCWaveFunctions/Fermion/MultiDiracDeterminant.cpp

```diff
@@ -425,6 +568,8 @@ void MultiDiracDeterminant::resize(int nel)
   new_grads.resize(NumDets, nel);
   lapls.resize(NumDets, nel);
   new_lapls.resize(NumDets, nel);
+  spingrads.resize(NumDets, nel);
+  new_spingrads.resize(NumDets, nel);
```

Comment:
resize here and above are not protected.

Reply:
I added a comment to the previous discussion we had about this above. I didn't push a fix yet because I wanted to check to see if my proposed solution would be adequate

Reply:
fixed